barman/infofile.py

```diff
@@ -767,6 +767,26 @@ def backup_type(self):
             return "rsync"
         return "incremental" if self.is_incremental else "full"
 
+    @property
+    def deduplication_ratio(self):
+        """
+        Returns a value between and including 0 and 1 related to the estimate
+        deduplication ratio of the backup.
+
+        .. note::
+            For `rsync` backups, the `size` of the backup, which is the sum of
+            all file sizes in basebackup directory, is used to calculate the
+            ratio. For `postgres` backups, the `cluster_size` is used.
```

Comment:
```suggestion
        .. note::
            For ``rsync`` backups, the :attr:`size` of the backup, which is the sum of
            all file sizes in basebackup directory, is used to calculate the
            ratio. For ``postgres`` backups, the :attr:`cluster_size` is used, which contains
            the estimated size of the Postgres cluster at backup time.
            
            We perform this calculation to make an estimation of how much network and disk
            I/O has been saved when taking an incremental backup through ``rsync`` or through
            ``pg_basebackup``.
            
            We abuse of the term "deduplication" here. It makes more sense to ``rsync`` than to
            ``postgres`` method. However, the idea is the same in both cases: get an estimation
            of resources saving.
```

Some fixes according to Sphinx recommendations, besides providing an extended context in the note.